config/environments/development.rb

```diff
@@ -11,6 +11,20 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Don't use this literally; use your local dev host instead
+  host = 'example.com'
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = {host: Settings.user.host }
```

Comment:
sử dụng khai báo biến host ở trên nha e
```suggestion
  config.action_mailer.default_url_options = {host: host, port: 3000}
```